main.py

```diff
@@ -1937,6 +1937,52 @@ async def random(message: discord.Interaction):
         except Exception:
             await message.followup.send("no cats :(")
 
+async def light_market(message):
+    cataine_prices = [[10, "Fine"], [30, "Fine"], [20, "Good"], [15, "Rare"], [20, "Wild"], [10, "Epic"], [20, "Sus"], [15, "Rickroll"],
+                      [7, "Superior"], [5, "Legendary"], [3, "8bit"], [4, "Professor"], [3, "Real"], [2, "Ultimate"], [1, "eGirl"]]
+    if get_cat(message.guild.id, message.user.id, "cataine_active") < int(time.time()):
+        count = get_cat(message.guild.id, message.user.id, "cataine_week")
+        embed = discord.Embed(title="The Mafia Hideout", description="you break down the door. the cataine machine lists what it needs.")
+
+        seed(datetime.datetime.utcnow().isocalendar()[1]) # hopefully that works
+        deals = []
+        for i in range(randint(2,4)) # 3-5 prices are possible per week
+            deals.append(cataine_prices[randInt(0,14)])
+        deals.sort()
```

Comment:
there is definitely a cleaner way to do this with list comprehensions and stuff but sure
why are we sorting here though?

Reply:
so that you get the cheapest deal first, and it goes up each cataine you buy during the week

Reply:
does it actually get cheaper though?

Reply:
more reasonabl-ish would be to instead just make a list of indexes, sort that and then access the list by those indexes when needed